### DISCUSSION TITLE
Update only 6.5" screenshots in iOS using Deliver

### DISCUSSION THREAD

**Discussion by Thangaraj-M** [fastlane/fastlane#17868] [Q&A]
Is there any option present in deliver lane to delete only 6.5" screenshots which already present and update with new 6.5" screenshots?

**Comment by rogerluan** [comment]
No 😅

**Reply by Thangaraj-M** [reply]
Is it possible to implement this in Fastlane?

**Reply by rogerluan** [reply]
Not sure if there's enough value to this. What I do today is replace all of them at once (the upload time is pretty quick, especially if you compress the images using something like tinypng). 

But contributions are always welcome, feel free to open a PR with your proposed changes 💪
